src/main/java/org/openstreetmap/atlas/geography/atlas/delta/AtlasDelta.java

```diff
@@ -36,28 +36,29 @@
  * Difference between two {@link Atlas}.
  *
  * @author matthieun
+ * @author hallahan
```

Comment:
Not sure a variable name change warrants an author tag ;)

Reply:
LOL, yeah, you're right! I'll take it out. I didn't realize I did it for that file.